5.0/en/0x51-V51-OAuth2.md

```diff
@@ -28,16 +28,10 @@ There are various different personas in the OAuth process, described in more det
 
 | # | Description | L1 | L2 | L3 |
 | :---: | :--- | :---: | :---: | :---: |
-| **51.3.1** | [ADDED] Verify that Resource Servers are using mechanisms for sender-constraining access tokens to prevent token replay, such as Mutual TLS for OAuth 2.0 or OAuth Demonstration of Proof of Possession (DPoP).  | ✓ | ✓ | ✓ |
+| **51.3.1** | [ADDED] Verify Resource Servers implement sender-constrained access token mechanisms to mitigate token replay risks. This is achieved by mandating Mutual TLS for OAuth 2.0 or OAuth Demonstration of Proof of Possession (DPoP), using the client secret provided at client registration. | ✓ | ✓ | ✓ |

```

Comment:
> "using the client secret provided"

I think that the use of "client secret" as a client authentication mechanism may pose security risks. Maybe we should consider recommending a secure client authentication mechanism?

Reply:
I think is is in response to Jim's point here:
https://github.com/OWASP/ASVS/pull/1804#issuecomment-1981022967

I am going to merge this for now @VinodAnandan but please do feel free to open a separate issue if you think this is still unclear.